app/services/plan_service/data_types.rb

```diff
@@ -18,6 +20,8 @@ module PlanService::DataTypes
     [:marketplace_plan_id, :fixnum, :mandatory],
     [:marketplace_id, :fixnum, :mandatory],
     [:plan_level, :fixnum, :mandatory],
+    [:features, :hash, :mandatory],
+    [:member_limit, :fixnum, :optional],
```

Comment:
Btw. if you decide to remove the `plan_level` from this entity, then the external plan service Entity type for this must be also changed.